static/src/javascripts/projects/commercial/modules/messenger/resize.js

```diff
@@ -44,18 +45,28 @@ const resize = (
     return fastdom.write(() => {
         Object.assign(adSlot.style, styles);
         Object.assign(iframe.style, styles);
+        Object.assign(iframeContainer.style, styles);
     });
 };
 
 const init = (register: RegisterListeners) => {
     register('resize', (specs, ret, iframe) => {
         if (iframe && specs) {
+            /* there are ultimately three containers that need resizing:
+                 1) the ad slot,
+                 2) the iframe container div (.ad-slot--content), and
+                 3) the iframe containing the creative
+             */
             const adSlot = iframe && iframe.closest('.js-ad-slot');
-            return resize(specs, iframe, adSlot);
+            const iframeContainer =
+                iframe && iframe.closest('.ad-slot__content');
+            return resize(specs, iframe, iframeContainer, adSlot);
         }
     });
 };
 
 export const _ = { resize, normalise };
 
 export { init };
+
+// #${iframe.id}__container__
```

Comment:
Why is this commented out?

Reply:
Ah I forgot to delete that line - originally I was determining the id of the iframe container but the CSS selector doesn't like ids with forward slashes in the name so I decided to just use the class instead.